src/lib.rs

```diff
@@ -282,6 +282,7 @@
 //! it will only happen in a minor or major version release.
 
 #![warn(missing_docs)]
+#![allow(renamed_and_removed_lints)]  // `missing_crate_level_docs` was renamed with a `rustdoc::` prefix
```

Comment:
Can we add a TODO here to update this when we bump our MSRV to a version that supports the new name?

Reply:
Done.